svf/lib/SVFIR/SVFIR.cpp

```diff
@@ -408,6 +476,7 @@ NodeID SVFIR::addGepValNode(NodeID curInst,const ValVar* baseVar, const AccessPa
            && "this node should not be created before");
     GepValObjMap[curInst][std::make_pair(base, ap)] = i;
     GepValVar *node = new GepValVar(baseVar, i, ap, type, icn);
+    node->setLLVMVarInstID(curInst);
```

Comment:
Why do we need to set the instruction here?

Reply:
the curInst is a nodeID rather than an instruction. we need to set this id here in order to maintain the GepValObjMap. the GepValObjMap use the curInst as the key. When we read this node from the database, because we don't have the bc, we don't what is the current instruction. so we need to store the id here, and store this instruction id to db.